CMakeLists.txt

```diff
@@ -705,7 +708,13 @@ if(HERMES_IS_ANDROID)
   find_package(fbjni REQUIRED CONFIG)
 endif()
 
-set(CMAKE_CXX_STANDARD 17)
+if("${CMAKE_CXX_COMPILER_ID}" MATCHES "MSVC")
+  # MSCV requires c++20
+  set(CMAKE_CXX_STANDARD 20)
+else()
+  set(CMAKE_CXX_STANDARD 17)
```

Comment:
React native build with Cxx 20 for everything.
Why do we need to keep building with 17?

Reply:
We generally don't upgrade Hermes' C++ standard requirements until we have a compelling reason to, so that we can maximise compatibility with different toolchains.